pkg/util/hlc/hlc.go

```diff
@@ -181,36 +182,46 @@ func (c *Clock) PhysicalTime() time.Time {
 	return timeutil.Unix(0, c.PhysicalNow())
 }
 
-// Update takes a hybrid timestamp, usually originating from
-// an event received from another member of a distributed
-// system. The clock is updated and the hybrid timestamp
-// associated to the receipt of the event returned.
-// An error may only occur if offset checking is active and
-// the remote timestamp was rejected due to clock offset,
-// in which case the timestamp of the clock will not have been
-// altered.
-// To timestamp events of local origin, use Now instead.
+// Update takes a hybrid timestamp, usually originating from an event
+// received from another member of a distributed system. The clock is
+// updated and the hybrid timestamp associated to the receipt of the
+// event returned. If the remote timestamp exceeds the wall clock
+// time by more than the maximum clock offset, the update is rejected
+// and a warning is logged.
```

Comment:
This comment is incorrect (it was incorrect before, though).

Reply:
Will send an update.